### ISSUE TITLE
Some songs won't play back

### ISSUE THREAD

**Issue by KingDuckZ** [Znurre/Subsoniq#14]
Some songs on my Airsonic server just won't start. My music is mostly vorbis files encoded from audio CD through Soundkonverter. I suspect that the problematic files are those for which I forgot to check the "Sample size" tick box in Soundkonverter options dialogue. Please let me know if you would like a sample.

**Comment by KingDuckZ** [comment]
I've got an update, I found why songs don't play correctly: my ogg files are almost all corrupt. Thanks to [this bug](https://trackitweb.com/fix-ogg-files-corrupted-by-easytag/) in EasyTag ogg and opus files get the beginning of the audio overwritten with invalid data. Most players, including clementine, mpv, DSub and Airsonic itself are just tolerant and skip the damaged part (I think that's the correct behaviour). Subsoniq is clearly lacking some sanity checks and just gets into a very unstable state. Possibly related to #16.

Needless to say, I've abandoned EasyTag in favour of Picard and I'm re-encoding my entire collection, so this particular problem will be fixed for me soon. Imho, it would still be nice to add some robustness in the face of corrupt input files.
